OpenEdXMobile/build.gradle

```diff
@@ -115,12 +113,12 @@ dependencies {
     // with module updates.
     implementation project (':android-iconify-fontawesome')
 
-    implementation "com.android.support:appcompat-v7:${SUPPORT_LIBRARY_VERSION}"
-    implementation "com.android.support:recyclerview-v7:${SUPPORT_LIBRARY_VERSION}"
-    implementation "com.android.support:cardview-v7:${SUPPORT_LIBRARY_VERSION}"
-    implementation "com.android.support:design:${SUPPORT_LIBRARY_VERSION}"
+    implementation "androidx.appcompat:appcompat:1.0.0"
+    implementation "androidx.recyclerview:recyclerview:1.0.0"
+    implementation "androidx.cardview:cardview:1.0.0"
+    implementation "com.google.android.material:material:1.0.0"
     // For the optional Nullable annotation
-    implementation "com.android.support:support-annotations:${SUPPORT_LIBRARY_VERSION}"
+    implementation "androidx.annotation:annotation:1.0.0"
```

Comment:
`SUPPORT_LIBRARY_VERSION` not used anymore, also we need to add a new constant to represent the androidX Library version.